src/app/components/Modal/styles.tsx

```diff
@@ -69,9 +58,39 @@ const Body = styled.div`
   }
 `;
 
+// tslint:disable-next-line:variable-name
+export const Mask = styled.div`
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.3);
+`;
+
+// tslint:disable-next-line:variable-name
+export const ModalWrapper = styled.div`
+  top: 0;
+  z-index: ${theme.zIndex.errorPopup};
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  position: fixed;
+  justify-content: center;
+  align-items: center;
+`;
+
+// tslint:disable-next-line:variable-name
+export const CardWrapper = styled.div`
+  z-index: 1;
+`;
+
 // tslint:disable-next-line:variable-name
 export const Footer = styled.div`
   display: flex;
+  justify-content: space-between;
```

Comment:
i worry about the assumptions you're making using this to arrange your buttons, especially since in reality someone implementing this component can pass arbitrary content with any number of children to the footer. for instance with 3 buttons this would look super weird. i think we should either:

- take this out, and just let multiple buttons be next to each other
- designate separate ways to specify buttons that go on the left vs on the right